website/content/docs/secrets/databases/oracle.mdx

```diff
@@ -1,37 +1,60 @@
 ---
 layout: docs
-page_title: Oracle - Database - Secrets Engines
+page_title: Oracle - database - secrets engines
 description: |-
   Oracle is one of the supported plugins for the database secrets engine. This
   plugin generates database credentials dynamically based on configured roles
   for the Oracle database.
 ---
 
-# Oracle Database Secrets Engine
+# Oracle database secrets engine
 
-This secrets engine is a part of the Database Secrets Engine. If you have not read the
-[Database Backend](/vault/docs/secrets/databases) page, please do so now as it explains how to set up the database backend and
+-> The Oracle database plugin is now available for use with the database secrets engine in HCP Vault.
```

Comment:
`with the database secrets engine in HCP Vault on all tiers`

Reply:
Resolved in upcoming commit